packages/paypal-commerce-integration/src/paypal-commerce-fastlane/paypal-commerce-fastlane-customer-strategy.ts

```diff
@@ -175,16 +175,24 @@ export default class PayPalCommerceFastlaneCustomerStrategy implements CustomerS
         }
 
         if (shippingAddress && cart.lineItems.physicalItems.length > 0) {
-            const consignments = state.getConsignments() || [];
+            await this.paymentIntegrationService.updateShippingAddress(shippingAddress);
+
+            const updatedState = this.paymentIntegrationService.getState();
```

Comment:
As far as I know `updateShippingAddress` method returns updated state, therefore this method can be simplified:
```suggestion
            const updatedState = await this.paymentIntegrationService.updateShippingAddress(shippingAddress);
```

P.s: It might be better to double check if its true, however TS interfaces said that it is

Reply:
fixed